src/boot.tsx

```diff
@@ -31,7 +47,8 @@ export default (element: HTMLDivElement, hub: Hub) => {
     const exercise = JSON.parse(atob(decodeURIComponent(element.textContent)));
     settings.hint = exercise.hint;
     settings.language = exercise.language;
-    settings.pre_exercise_code = exercise.pre_exercise_code;
+    settings.lang_version = exercise.lang_version;
+    settings.pre_exercise_code = exercise.pre_exercise_code + getPackages(exercise.packages);
```

Comment:
I think you want to do the `getPackages` before the `pre_exercise_code` this way you can use any of the installed modules in the pec.

Reply:
True, good idea